### PR TITLE
Removed mocha-eslint and eslint-config-node-services dependencies.

### PR DESCRIPTION
**Pull request by hknustwmf**

### CODE REVIEW

File: .eslintrc.yml [hunk 1]
```diff
@@ -1 +1,31 @@
-extends: 'eslint-config-node-services'
\ No newline at end of file
+extends: 
+  - 'eslint-config-wikimedia/server'
```
Comment (jdforrester) [inline .eslintrc.yml]: This should work as just 'wikimedia/server' unless something's broken.

Comment (Pchelolo) [reply .eslintrc.yml]: +1

File: .eslintrc.yml [hunk 2]
```diff
@@ -1 +1,31 @@
...
+rules:
+  array-bracket-spacing:
+    - off
+  camelcase:
+    - error
+    - properties: never
+  computed-property-spacing:
+    - off
+  indent: 
+    - off
```
Comment (Pchelolo) [inline .eslintrc.yml]: Actually, why is it off? We do want to validate indent, just want to override tabs to spaces.

File: package.json [hunk 3]
```diff
@@ -47,18 +48,15 @@
     "semver": "latest"
   },
   "devDependencies": {
-    "ajv": "^6.5.4",
+    "ajv": "^6.7.0",
```
Comment (Pchelolo) [inline package.json]: yet again, let's not update unrelated dependencies.

Comment (hknustwmf) [reply package.json]: I updated only dev dependencies to bring them up to date. Did not deem that dangerous. I'll roll them back.

File: .eslintrc.yml (hunk 2 above)
Comment (hknustwmf) [reply .eslintrc.yml]: https://github.com/wikimedia/change-propagation/pull/302#pullrequestreview-194255471

File: .eslintrc.yml (hunk 1 above)
Comment (hknustwmf) [reply .eslintrc.yml]: No, it works with 'wikimedia/server', Going to update that.

File: maintenance/generate-wikimedia-domain-config.js [hunk 4]
```diff
@@ -31,11 +27,11 @@ preq.get({
         '*': []
     };
 
-    Object.keys(sm).forEach(function(k) {
+    Object.keys(sm).forEach(function (k) {
```
Comment (d00rman) [inline maintenance/generate-wikimedia-domain-config.js]: Maybe turn this and the ones below into `() => {}`

File: .eslintrc.yml (hunk 2 above)
Comment (d00rman) [reply .eslintrc.yml]: This is different from [the one in `rb-m-t-spec`](https://github.com/wikimedia/restbase-mod-table-spec/pull/53/files#diff-0378d5111ab996d5882f4712e9350604R13). While both have the same effect, we should probably use the same values to minimise confusion.

**Comment by berndsi** [comment]
Indentation looks good to me.

The only thing I would change is 
"operator-linebreak": [ "error", "after" ],
to 
"operator-linebreak": [ "error", "before" ],
since it's easier to see an operator at the beginning of the line than at the end, so one quickly sees that the line is a continuation.

**Comment by d00rman** [comment]
> since it's easier to see an operator at the beginning of the line than at the end, so one quickly sees that the line is a continuation.

I tangentially agree, but we've gone with the former everywhere else, so let's keep it consistent.

**Pull request merged**